README.ABI

```diff
@@ -1,4 +1,4 @@
-When using gnu-efi 4.0 with an application which was wrote
+(1) When using gnu-efi 4.0 with an application which was wrote
```

Comment:
Might fix the typo too: "wrote" &rarr; "written"

Reply:
Done